openy_gated_content.libraries.yml

```diff
@@ -1,3 +1,9 @@
+styles:
```

Comment:
let's rename lib to `admin-ui-styles`

Reply:
Also, I can't see the place where this lib attached

Reply:
It was attached earlier -- just the library declaration was removed someday :)